# RPY message - roll-pitch-yaw-throttle signals
Header header
float32 delta_a
float32 delta_e
float32 delta_r
float32 delta_t
